data/transport missions.txt

```diff
@@ -1975,3 +1975,44 @@ ship "Science Drone"
 		"heat dissipation" 1
 	explode "tiny explosion" 5
 	explode "small explosion" 2
+
+
+
+mission "Loan Broker Reunion"
+	name "Transfer enforcers to <planet>"
+	description "Transport 15 enforcers from <source> to <destination> by <day> to fulfill a request from the New Boston loan broker."
+	deadline
+	passengers 15
+	minor
+	to offer
+		random < 25
+		"unpaid mortgages" == 0
+	source "New Boston"
+	destination
+		government "Syndicate"
+	on offer
+		conversation
+			`As you're walking through the spaceport searching for a bar or pub, you notice one person who stands out. A nearly bald, middle-aged man wanders through the crowd in an expensively tailored suit, searching the faces that pass by him. You pause and look at him more closely; he seems oddly familar. Eventually, his gaze settles on you, making eye contact. He stares, then begins moving through the crowd toward you.`
```

Comment:
It's not very plausible to me that this fellow would recognize you. If he's a loan broken, surely he's loaned money to thousands of people, right? Maybe it's the player who should recognize _him!_

Reply:
Thanks for the feedback. I've edited some of the dialogue further down to clarify that the broker's looking for you in the spaceport and has a picture, thus giving him a reason to recognize you.